e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/currency-of-agents-charge/change-your-answers-currency-of-agents-charge-alternative-currency.spec.js

```diff
@@ -0,0 +1,108 @@
+import { summaryList } from '../../../../../../../pages/shared';
+import { INSURANCE_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import application from '../../../../../../../fixtures/application';
+import { EUR, NON_STANDARD_CURRENCY_CODE, NON_STANDARD_CURRENCY_NAME, SYMBOLS } from '../../../../../../../fixtures/currencies';
+import formatCurrency from '../../../../../../../helpers/format-currency';
+
+const {
+  ROOT,
+  EXPORT_CONTRACT: { CHECK_YOUR_ANSWERS },
+} = INSURANCE_ROUTES;
+
+const {
+  CURRENCY: { CURRENCY_CODE },
+  EXPORT_CONTRACT: {
+    AGENT_CHARGES: { FIXED_SUM_AMOUNT },
+  },
+} = INSURANCE_FIELD_IDS;
+
+const currencyAssertion = {
```

Comment:
could this be assertion and we pass it the fieldId as a variable?
`const assertion = (fieldId) => {
  row: summaryList.field(fieldId),
  expected: '',
}`

Reply:
Why? We only change the `expected` value and is only used in the "change agent charges" tests. I.e, after constructing the `assertion` object we only do this 2-4 times later in the test:

```ts
currencyAssertion.expected = someOtherCurrency
currencyAssertion.expected = aDifferentCurrency
amountAssertion.expected = newAmount
```

etc